runtime/compiler/p/codegen/J9PPCSnippet.cpp

```diff
@@ -1311,13 +1311,13 @@ void TR::createCCPreLoadedCode(uint8_t *CCPreLoadedCodeBase, uint8_t *CCPreLoade
 uint8_t *TR::PPCAllocPrefetchSnippet::emitSnippetBody()
    {
    TR::Compilation *comp = cg()->comp();
+   if (comp->getOptions()->realTimeGC())
```

Comment:
You moved the return statement up. Is it possible that the code executed before this return needed to stay because it had important side effects?
```
getSnippetLabel()->setCodeLocation(buffer);
```

Reply:
Will move it back to where it was.